### PR TITLE
Library documentation

### PR DESCRIPTION
**Pull request by Exulansis**
Addresses #203.

### CODE REVIEW

File: docs/gettingStarted.rst [hunk 1]
```diff
@@ -1,77 +1,85 @@
...
-we need to derive at least two keys. The first one is the identity key which is used for signing.
-The second one is an Ethereum key which is used for registering the identity on the Ethereum
-blockchain.  
+The ``identityManager`` instance we have created can now be used to derive further key pairs needed to complete the registration.
+Firstly, we need to derive the key pair that will be used to control your Jolocom Identity.
 
 .. code-block:: typescript
 
+  const identityKeyDerivationPath = identityManager.getSchema().jolocomIdentityKey // Derivation path - 'm/73'/0'/0'/0'
   const identityKey = identityManager.deriveChildKey(path)
```
Comment (nataschaberg) [inline docs/gettingStarted.rst]: I think here you want to pass  ``identityKeyDerivationPath`` to ``identityManager.deriveChildKey()``.

File: docs/gettingStarted.rst [hunk 2]
```diff
@@ -1,77 +1,85 @@
...
-``deriveChildKey`` on ``identityManager`` is called with ``path``. Path is a *string* and is a concept explored in previously mentioned  BIP 32 -
-it indicates along which 'path' a child key is derived from the master key. The Jolocom library provides
-default key paths which should be used during the creation process. You can use them like so:
+The only argument we need to pass to ``identityManager.deriveChildKey`` is the derivation ``path``, as defined in `BIP-32 <https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki>`_.
+The Jolocom library ships with a number of predefined paths for generating specific key pairs, which can be accessed as follows:
 
 .. code-block:: typescript
 
-  const schema = identityManager.getSchema()
+  // identityManager.getScehma() returns all paths shipped with the library
```
Comment (katszwn) [inline docs/gettingStarted.rst]: There's a typo

File: docs/gettingStarted.rst [hunk 3]
```diff
@@ -1,77 +1,85 @@
...
+  //  {
+  //    jolocomIdentityKey: "m73'/0'/0'/0'",
+  //    ethereumKey: "m/44'/60'/0'/0/0'"
+  //  }
```
Comment (nataschaberg) [inline docs/gettingStarted.rst]: I think the section above should be removed.

File: docs/gettingStarted.rst [hunk 4]
```diff
@@ -81,59 +89,91 @@ default key paths which should be used during the creation process. You can use
...
+  })
 
-Now you can use the ``registry`` for the last steps of identity creation and registration.
+.. warning:: You might observe, 2 private keys are needed to create an identity. The first key, ``privateIdentityKey`` is the one that will be used to control your Jolocom identity.
+  The second key, ``privateEthereumKey`` is only used to broadcast the identity creation transaction to the Ethereum network. Due to this, the ``privateEthereumKey``
+  should have enough Rinkeby ether associated with it to pay for the identity creation.
+  In the close future, the ``privateEthereumKey`` will be deprecated in favour of executable signed messages as defined in `EIP-1077 <https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1077.md>`_.
+
+.. seealso:: In case you are looking for a easy way to receive some Rinkeby Ether for testing purposes, all you need to do is send a ``POST`` request with your Ethereum address to the `corresponding endpoint <https://faucet.jolocom.com/request/>`_.
+  Reference implementation can also be found `here <https://github.com/jolocom/smartwallet-app/blob/develop/src/lib/ethereum.ts#L21>`_.
```
Comment (katszwn) [inline docs/gettingStarted.rst]: 👌

File: docs/interactionFlows.rst [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+
+  if (signatureValidationResults.every(result => result === true)) {
+    // The credentials can be used
+  }
+
+What next?
+###########
+
+Now that we understand the reasoning behind the credential request and response flows, it's time to test them out in action!
+Head to the next section to learn how to set up your own serbice so it can interact with Jolocom identities.
```
Comment (nataschaberg) [inline docs/interactionFlows.rst]: typo, should be service

File: docs/signedCredentials.rst [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+
+In order to create a signed credential, we need an instance of an ``identityWallet``. We can create a new one as defined in the `getting started <https://jolocom-lib.readthedocs.io/en/latest/gettingStarted.html>`_ section.
+
+In case you've already created an identity, you can get an instance of an ``identityWallet`` as follows:
+
+.. code-block:: typescript
+
+  import { JolocomLib } from 'jolocom-lib'
+  import { privIdentityKeyHex } from '..'
+  import { claimsMetadata } from 'cred-types-jolocom-core'
```
Comment (katszwn) [inline docs/signedCredentials.rst]: Maybe we could include some more information on the credential types from this repo?

**Comment by nataschaberg** [comment]
👍

**Pull request merged**